src/main/java/xyz/aadev/generitech/client/gui/power/GuiGenerator.java

```diff
@@ -61,6 +62,9 @@ public void drawBG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
 
     }
 
+
+
+
     @Override
     public void drawFG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1186)